code/drasil-build/lib/Build/Drasil/Make/AST.hs

```diff
@@ -7,8 +7,8 @@ import Build.Drasil.Make.MakeString (MakeString)
 -- | A Makefile is made up of Makefile rules.
 newtype Makefile = M [Rule]
 
--- | A Makefile Rule needs a target, dependencies, type, and commands.
-data Rule = R Target Dependencies Type [Command]
+-- | A Makefile Rule can have comments and commands, but needs a target, dependencies, a type, and commands.
```

Comment:
This now says "A ... Rule can have ... commands, but needs ... commands." One of the instances of "commands" should be removed depending on if they are optional or not 😅

Reply:
My bad! 😅 I will change that right now!